test/addons/external-snapshotter/crds/kustomization.yaml

```diff
@@ -6,3 +6,6 @@ resources:
   - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-8.2/client/config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml
   - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-8.2/client/config/crd/snapshot.storage.k8s.io_volumesnapshotcontents.yaml
   - https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/release-8.2/client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml
+  - https://raw.githubusercontent.com/red-hat-storage/external-snapshotter/refs/heads/master/config/crd/bases/groupsnapshot.storage.openshift.io_volumegroupsnapshotclasses.yaml
+  - https://raw.githubusercontent.com/red-hat-storage/external-snapshotter/refs/heads/master/config/crd/bases/groupsnapshot.storage.openshift.io_volumegroupsnapshotcontents.yaml
+  - https://raw.githubusercontent.com/red-hat-storage/external-snapshotter/refs/heads/master/config/crd/bases/groupsnapshot.storage.openshift.io_volumegroupsnapshots.yaml
```

Comment:
Who reconciles the private crds? Are they really part of external snapshotter or should be in a new addon?

Reply:
The new private api are being used in ramen now. And drenv needs these crds. There is a pr which is blocked because of this. E2E tests are failing. This is pr https://github.com/RamenDR/ramen/pull/2030.

Reply:
We need to find the right place to add the new crds and/or controller.